integration_test/README.md

```diff
@@ -0,0 +1,95 @@
+# Integration Tests
+
+Integration tests can be run in develop (local) mode or in CI environment for validating VAL components in Dapr/Kubernetes environment.\
+SeatService is running in simulated CAN mode `CAN="cansim"` and SeatService client is used to start seat movement.
+
+## Integration Test overview
+
+- `integration_test/test_feeder.py`:\
+    This test covers feedercan datapoints. It checks if they are registered and also that some datapoints have changing values.\
+    **NOTE:** If feedercan default config is changed, make sure those changes are reflected in feeder test.
+
+- `integration_test/test_val_seat.py`:\
+    This test covers seat service metadata and several seat move scenarios.\
+    It uses an external script for asking the seat to move to desired position (using `seat_svc_client` as grpc client)
+
+- `integration_test/broker_subscriber.py`:\
+Databroker grpc subscription handler (useful for scripting)
+
+  ```text
+    Usage: ./broker_subscriber.py --addr <host:name> [ --get-meta=META | --query <QUERY> --timeout <TIMEOUT> --count <COUNT> ]
+
+    Environment:
+      'BROKER_ADDR' Default: localhost:55555
+      'QUERY'       SQL datapoint query. ('*' = subscribe for all meta). Default: SELECT Vehicle.Cabin.Seat.Row1.Pos1.Position
+      'COUNT'       Receive specified count of events and exit (0=inf)
+      'TIMEOUT'     Abort receiving if no data comes for specified timeout in seconds (0=inf)
+      'META'        Comma separated list of datapoint names to query. ('*' = all meta)
+  ```
```

Comment:
Move to kuksa.val repo (data broker)?

Reply:
It is a helper script, focused on integration testing (manual or scripting).
Not sure if It fits in databroker releases / integration testing, but if moved I need a way to get it in this repo.